### PR TITLE
Fix build with Windows/MSVC and USD v23.11

### PR DESCRIPTION
**Pull request by pablode**
The changes in this PR should retain compatibility with macOS and Linux.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -7,9 +7,7 @@ include(GNUInstallDirs)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
-
-find_package(USD REQUIRED)
+find_package(USD REQUIRED NAMES pxr)
```
Comment (raph080) [inline CMakeLists.txt]: Interesting ! Could we improve it to `find_package(pxr REQUIRED)` ?

File: CMakeLists.txt [hunk 2]
```diff
@@ -7,9 +7,7 @@ include(GNUInstallDirs)
...
 find_package(Python COMPONENTS Interpreter Development REQUIRED)
```
Comment (raph080) [inline CMakeLists.txt]: If I understand well, now Python is not required anymore, so we can remove it ?

**Review by raph080 — COMMENTED** [review]
Hey @pablode,
Thank you very much for your pull request and your fixes.

Also, your [Rework USD CMake inclusion](https://github.com/raph080/ImGuiHydraEditor/pull/1/commits/d2c20354db102b402b60ae72cabce6c6acd13cf9) was a very interesting commit. I did not know about the pxrConfig.cmake that could be loaded using some -Dpxr_DIR. Thanks for that !

File: CMakeLists.txt (hunk 1 above)
Comment (pablode) [reply CMakeLists.txt]: Seems to work! Also does not require `-Dpxr_dir` anymore. But: expects USD and its dependencies to be in the PATH (which should be the case for a proper USD installation).

File: CMakeLists.txt (hunk 2 above)
Comment (pablode) [reply CMakeLists.txt]: Indeed, I've removed it.

**Comment by raph080** [comment]
Thank you for that last clean up !

**Pull request merged**